docs/LLS_Metallicities-Wotta/mcmc_met.py

```diff
@@ -0,0 +1,258 @@
+"""

```

Comment:
It was absorbed into `pyigm_mtlmcmc.py` as per our conversation at Durham.  There is a new `--wotta` option in there that allows you to read in the guesses file, then passes that off to the preexisting function.